test/unit/model/qti/datatype/LanguageValidateLanguageCodesTest.php

```diff
@@ -1,24 +1,99 @@
 <?php
 
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2023 (original work) Open Assessment Technologies SA.
+ */
+
 namespace oat\taoQtiItem\test\unit\model\qti\datatype;
 
-use oat\generis\test\TestCase;
+use PHPUnit\Framework\TestCase;
 use oat\taoQtiItem\model\qti\datatype\Language;
 
 class LanguageValidateLanguageCodesTest extends TestCase
 {
     private const LANGUAGE_CODES_JSON_PATH = __DIR__ . '/../../../../resources/pisa-qa-translation-locales.json';
 
     /**
-     * @dataProvider provideLanguageCodes
+     * @dataProvider validPisaLanguageCodesDataProvider
+     * @dataProvider validTwoLetterCodes
+     * @dataProvider invalidLanguageCodesDataProvider
      */
-    public function testValidate(string $languageCode): void
+    public function testValidate(bool $expected, string $languageCode): void
```

Comment:
Lets add all the PISA languages as payload to make sure this will cover all their cases, please.

Reply:
We already have all of them here: https://github.com/oat-sa/extension-tao-itemqti/blob/84da92ffb746d50a4c386b89b48c3fced60811eb/test/resources/pisa-qa-translation-locales.json

This file contains all the codes from the translation-locales.json file available in the PISA QA bucket.

Reply:
Got it, thanks. It is not related to your task, but could you please adjust that file to only contain the `language code` list, as we do not really care about other data?

Also, maybe can we remove the PISA specific naming and instead just rename that file to `supported-languages`?

And the content would be something like: 

```
[
   {
      "code": "ar-ARB",
      "title": "Arabic (ARB)"
    },
]
```

Reply:
Changed in 13712168314e5129f4609db8dbd03ebb208ab135 & aca176ae0cd5b967740c28aebfd574147bb7aca4

Reply:
Thanks!